src/datasets/once/once_frame_patcher.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+import numpy as np
+
+from src.datasets.once.once_utils import ONCE
+from src.datasets.frame_patcher import FramePatcher
+from src.datasets.once.once_utils import get_frame_point_cloud, reapply_frame_transformation, get_frame_instance_ids
+from src.utils.geometry_utils import points_in_box
+
+
+class OnceFramePatcher(FramePatcher):
+    """Patches Once frames with new point clouds.
+    """
+
+    def __init__(self,
+                 seq_id: str,
+                 frame_id: str,
+                 frame_point_cloud: np.ndarray,
+                 # frame_descriptor: dict,
+                 once: ONCE):
+        self.__seq_id = seq_id
+        self.__frame_id = frame_id
+        self.__frame_point_cloud = frame_point_cloud
+        # self.__frame_descriptor = frame_descriptor

```

Comment:
Can you, please, remove the commented lines here and in other places in the codebase?

Reply:
removed